src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -593,7 +593,12 @@ public function install_package( $args = array() ) {
 		}
 
 		// Copy new version of item into place.
-		$result = copy_dir( $source, $remote_destination );
+		if ( class_exists( 'Rollback_Update_Failure\WP_Upgrader' ) && function_exists( '\Rollback_Update_Failure\move_dir' )
+		) {
```

Comment:
```suggestion
		if ( class_exists( 'Rollback_Update_Failure\WP_Upgrader' ) && function_exists( '\Rollback_Update_Failure\move_dir' ) ) {
```

Either use a multi-line statement or truly use a single-line statement, this is sort of a mix of both and not allowed.

Reply:
oops. fixed.